packages/commerce-sdk-react/src/hooks/ShopperSeo/index.ts

```diff
@@ -0,0 +1,8 @@
+/*
+ * Copyright (c) 2022, Salesforce, Inc.
```

Comment:
```suggestion
 * Copyright (c) 2024, Salesforce, Inc.
```

Reply:
You might want to check the other files too

Reply:
You still missed this one.

Reply:
@bendvc it has been updated last Friday. you may be on old cold